js/scenes/Home.js

```diff
@@ -38,6 +38,7 @@ export class Home extends Component {
   componentDidMount() {
     const { dispatch } = this.props;
     dispatch(fetchItemsIfNeeded());
+    dispatch(watchGeoLocation());
```

Comment:
Actually I was thinking on not to stop watching while the user is on the app. We will need to keep updating location to have proper pointer to the compasses.

@wilsonpage what's your view?